cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,63 @@
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import faker from 'faker';
+
 /// <reference types="cypress" />
 
-describe('', () => {
-  before(() => {
+const homePage = new HomeAndCataloguePageObject();
 
-  });
 
-  it('', () => {
+const testData = {
+  category: 'Laptops',
+  product: 'Sony vaio i7',
+  name: faker.name.firstName(),
+  country: faker.address.country(),
+  city: faker.address.city(),
+  cardNumber: faker.finance.creditCardNumber(),
+  month: faker.date.month(),
+  year: Math.floor(Math.random() * (2031 - 2023) + 2023),
+  successMessage: 'Thank you for your purchase'
+};
+
+describe('Order', () => {
+  before(() => {
+    homePage.visit();
+  });
 
+  it('User is able to order the product', () => {
```

Comment:
you don't use POM in your tests. Check the [theory](https://mate.academy/en/learn/javascript-testing/cypress?section=theory&videoId=2134&testTaskSlug=qa_cypress_demoblaze_pom_basics)